docsrc/source/pages/use_cases/comparing_datasets.rst

```diff
@@ -2,8 +2,6 @@
 Dataset Comparison
 ==================
 
-*This feature was introduced in pandas-profiling 3.4.*
```

Comment:
Why would this be removed? Python's docs note which version adds each new function, and it's easy to accidentally be reading docs for a version more recent than what you're using.

Reply:
This is not the correct version of the feature. Rest assured that by the time the release process version information will be included in the docs.